msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2019-02-17 02:24+0000\n"
"PO-Revision-Date: 2019-02-25 03:07\n"
"Last-Translator: Kowlin <boxedpp@gmail.com>\n"
"Language-Team: German\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 2.2\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Language: de\n"
"X-Crowdin-File: /cogs/warnings/locales/messages.pot\n"
"Language: de_DE\n"

#: redbot/cogs/warnings/helpers.py:70
msgid "I could not find a command from that input!"
msgstr "Ich konnte keinen Befehl für diese Eingabe finden!"

#: redbot/cogs/warnings/helpers.py:77
msgid "That command requires bot owner. I can't allow you to use that for an action"
msgstr "Dieser Befehl kann nur vom Bot Besitzer ausgeführt werden. Du darfst diese Aktion nicht ausführen."

#: redbot/cogs/warnings/helpers.py:86
msgid "Enter the command to be run when the user **exceeds the points for this action to occur.**\n"
"**If you do not wish to have a command run, enter** `none`.\n\n"
"Enter it exactly as you would if you were actually trying to run the command, except don't put a prefix and use `{user}` in place of any user/member arguments\n\n"
"WARNING: The command entered will be run without regard to checks or cooldowns. Commands requiring bot owner are not allowed for security reasons.\n\n"
"Please wait 15 seconds before entering your response."
msgstr ""

#: redbot/cogs/warnings/helpers.py:99 redbot/cogs/warnings/helpers.py:143
msgid "You may enter your response now."
msgstr "Du kannst deine Antwort nun eingeben."

#: redbot/cogs/warnings/helpers.py:128
msgid "Enter the command to be run when the user **returns to a value below the points for this action to occur.** Please note that this is intended to be used for reversal of the action taken when the user exceeded the action's point value.\n"
"**If you do not wish to have a command run on dropping points, enter** `none`.\n\n"
"Enter it exactly as you would if you were actually trying to run the command, except don't put a prefix and use `{user}` in place of any user/member arguments\n\n"
"WARNING: The command entered will be run without regard to checks or cooldowns. Commands requiring bot owner are not allowed for security reasons.\n\n"
"Please wait 15 seconds before entering your response."
msgstr ""

#: redbot/cogs/warnings/warnings.py:26
#, docstring
msgid "Warn misbehaving users and take automated actions."
msgstr ""

#: redbot/cogs/warnings/warnings.py:51
#, docstring
msgid "Manage settings for Warnings."
msgstr ""

#: redbot/cogs/warnings/warnings.py:57
#, docstring
msgid "Enable or disable custom reasons for a warning."
msgstr ""

#: redbot/cogs/warnings/warnings.py:61
msgid "Custom reasons have been enabled."
msgstr "Benutzerdefinierte Gründe wurden aktiviert."

#: redbot/cogs/warnings/warnings.py:63
msgid "Custom reasons have been disabled."
msgstr "Benutzerdefinierte Gründe wurden deaktiviert."

#: redbot/cogs/warnings/warnings.py:69
#, docstring
msgid "Manage automated actions for Warnings.\n\n"
"        Actions are essentially command macros. Any command can be run\n"
"        when the action is initially triggered, and/or when the action\n"
"        is lifted.\n\n"
"        Actions must be given a name and a points threshold. When a\n"
"        user is warned enough so that their points go over this\n"
"        threshold, the action will be executed.\n"
"        "
msgstr ""

#: redbot/cogs/warnings/warnings.py:84
#, docstring
msgid "Create an automated action.\n\n"
"        Duplicate action names are not allowed.\n"
"        "
msgstr ""

#: redbot/cogs/warnings/warnings.py:105
msgid "Duplicate action name found!"
msgstr "Doppelter Aktionsname gefunden!"

#: redbot/cogs/warnings/warnings.py:112
msgid "Action {name} has been added."
msgstr ""

#: redbot/cogs/warnings/warnings.py:117
#, docstring
msgid "Delete the action with the specified name."
msgstr ""

#: redbot/cogs/warnings/warnings.py:130
msgid "No action named {name} exists!"
msgstr ""

#: redbot/cogs/warnings/warnings.py:136
#, docstring
msgid "Manage warning reasons.\n\n"
"        Reasons must be given a name, description and points value. The\n"
"        name of the reason must be given when a user is warned.\n"
"        "
msgstr ""

#: redbot/cogs/warnings/warnings.py:148
#, docstring
msgid "Create a warning reason."
msgstr ""

#: redbot/cogs/warnings/warnings.py:152
msgid "*Custom* cannot be used as a reason name!"
msgstr ""

#: redbot/cogs/warnings/warnings.py:162
msgid "The new reason has been registered."
msgstr "Der neue Grund wurde registriert."

#: redbot/cogs/warnings/warnings.py:167
#, docstring
msgid "Delete a warning reason."
msgstr "Lösche einen Warngrund."

#: redbot/cogs/warnings/warnings.py:174
msgid "That is not a registered reason name."
msgstr "Das ist kein registrierter Grund."

#: redbot/cogs/warnings/warnings.py:180
#, docstring
msgid "List all configured reasons for Warnings."
msgstr "Liste alle konfigurierten Gründe für Warnungen auf."

#: redbot/cogs/warnings/warnings.py:188
msgid "Reason: {name}"
msgstr "Grund: {name}"

#: redbot/cogs/warnings/warnings.py:190 redbot/cogs/warnings/warnings.py:215
#: redbot/cogs/warnings/warnings.py:297
msgid "Points"
msgstr "Punkte"

#: redbot/cogs/warnings/warnings.py:194
msgid "Name: {reason_name}\n"
"Points: {points}\n"
"Description: {description}"
msgstr "Name: {reason_name}\n"
"Punkte: {points}\n"
"Beschreibung: {description}"

#: redbot/cogs/warnings/warnings.py:201
msgid "There are no reasons configured!"
msgstr "Es sind keine Gründe konfiguriert!"

#: redbot/cogs/warnings/warnings.py:207
#, docstring
msgid "List all configured automated actions for Warnings."
msgstr "Liste alle konfigurierten automatisierten Aktionen für Warnungen auf."

#: redbot/cogs/warnings/warnings.py:214
msgid "Action: {name}"
msgstr "Aktion: {name}"

#: redbot/cogs/warnings/warnings.py:216
msgid "Exceed command"
msgstr "Befehl überschreiten"

#: redbot/cogs/warnings/warnings.py:217
msgid "Drop command"
msgstr "Befehl ablegen"

#: redbot/cogs/warnings/warnings.py:221
msgid "Name: {action_name}\n"
"Points: {points}\n"
"Exceed command: {exceed_command}\n"
"Drop command: {drop_command}"
msgstr "Name: {action_name}\n"
"Punkte: {points}\n"
"Befehl überschreiten: {exceed_command}\n"
"Befehl ablegen: {drop_command}"

#: redbot/cogs/warnings/warnings.py:229
msgid "There are no actions configured!"
msgstr "Es sind keine Aktionen konfiguriert!"

#: redbot/cogs/warnings/warnings.py:235
#, docstring
msgid "Warn the user for the specified reason.\n\n"
"        `<reason>` must be a registered reason name, or *custom* if\n"
"        custom reasons are enabled.\n"
"        "
msgstr "Warne den Benutzer aus dem angegebenen Grund.\n\n"
"        `<reason>` muss ein registrierter Grund-Name sein, oder eine *benutzerdefinierte*, wenn\n"
"        benutzerdefinierte Gründe aktiviert sind.\n"
"        "

#: redbot/cogs/warnings/warnings.py:241
msgid "You cannot warn yourself."
msgstr "Du kannst dich selbst nicht verwarnen."

#: redbot/cogs/warnings/warnings.py:247
msgid "Custom reasons are not allowed! Please see `{prefix}reasonlist` for a complete list of valid reasons."
msgstr "Benutzerdefinierte Gründe sind nicht erlaubt! Siehe `{prefix}reasonlist` für eine komplette Liste von gültigen Gründen."

#: redbot/cogs/warnings/warnings.py:258
msgid "That is not a registered reason!"
msgstr "Das ist kein registrierter Grund!"

#: redbot/cogs/warnings/warnings.py:260
msgid "Do `{prefix}warn {user} custom` to specify a custom reason."
msgstr "Nutze `{prefix}warn {user} custom`, um einen benutzerdefinierten Grund anzugeben."

#: redbot/cogs/warnings/warnings.py:268
msgid "Do `{prefix}warningset allowcustomreasons true` to enable custom reasons."
msgstr "Nutze `{prefix}warningset allowcustomreasons true`, um benutzerdefinierte Gründe zu aktivieren."

#: redbot/cogs/warnings/warnings.py:294
msgid "Warning from {user}"
msgstr "Warnung von {user}"

#: redbot/cogs/warnings/warnings.py:299
msgid "You have received a warning in {guild_name}."
msgstr "Du hast eine Warnung in {guild_name} erhalten."

#: redbot/cogs/warnings/warnings.py:306
msgid "User {user} has been warned."
msgstr "Der Benutzer {user} wurde verwarnt."

#: redbot/cogs/warnings/warnings.py:313
#, docstring
msgid "List the warnings for the specified user.\n\n"
"        Omit `<user>` to see your own warnings.\n\n"
"        Note that showing warnings for users other than yourself requires\n"
"        appropriate permissions.\n"
"        "
msgstr ""

#: redbot/cogs/warnings/warnings.py:325
msgid "You are not allowed to check warnings for other users!"
msgstr "Du bist nicht berechtigt Warnungen anderer Benutzer zu überprüfen!"

#: redbot/cogs/warnings/warnings.py:339
msgid "That user has no warnings!"
msgstr "Dieser Benutzer hat noch keine Verwarnungen!"

#: redbot/cogs/warnings/warnings.py:349
msgid "{num_points} point warning {reason_name} issued by {user} for {description}\n"
msgstr ""

#: redbot/cogs/warnings/warnings.py:359
msgid "Warnings for {user}"
msgstr "Warnung für {user}"

#: redbot/cogs/warnings/warnings.py:366
#, docstring
msgid "Remove a warning from a user."
msgstr "Entferne eine Warnung von einem Benutzer."

#: redbot/cogs/warnings/warnings.py:379
msgid "You cannot remove warnings from yourself."
msgstr "Du kannst keine Warnungen von dir selbst entfernen."

#: redbot/cogs/warnings/warnings.py:386
msgid "That warning doesn't exist!"
msgstr "Diese Warnung existiert nicht!"

#: redbot/cogs/warnings/warnings.py:398
msgid "How many points should be given for this reason?"
msgstr "Wie viele Punkte sollten für diesen Grund gegeben werden?"

#: redbot/cogs/warnings/warnings.py:404 redbot/cogs/warnings/warnings.py:423
msgid "Ok then."
msgstr "Ok."

#: redbot/cogs/warnings/warnings.py:409
msgid "That isn't a number!"
msgstr "Wert ist keine Zahl!"

#: redbot/cogs/warnings/warnings.py:413
msgid "The point value needs to be greater than 0!"
msgstr "Der Wert muss größer als 0 sein!"

#: redbot/cogs/warnings/warnings.py:417
msgid "Enter a description for this reason."
msgstr "Gib eine Beschreibung für diesen Grund an."

